app/views/instances/_show_waiting.erb

```diff
@@ -60,12 +56,14 @@
       <p>Instance pin:</p>
       <p><%= instance.pin %></p>
     </div>
+    <% if current_user.present? && current_user.id == @instance.user_id %>
```

Comment:
```suggestion
    <% if current_user.id == @instance.user_id %>
```
No need to check if current_user is present as we don't allow unsigned users anyway :)